README.md

```diff
@@ -0,0 +1,123 @@
+# API Specification Version Maven Plugin
+
+The API Specification Version Plugin provides goals to set properties for manifest file OSGI headers and to verify consistence of specification, API and API implementation package names and versions.
+
+**Goals:**
+* `set-spec-properties` (validate phase)
+* `check-module` (package phase)
+
+## Configuration properties
+
+| Property | Type | Default<br/>Value | Description |
```

Comment:
I would have a different section for each goal.

Reply:
Done, spec was left as whole object.